File: cal.c
/* Basic Calculator in C 

Operations supported:
1. Addition

*/
#include <stdio.h>

int main(int argc, char *argv[])
{
  float a,b;

  printf("Enter two numbers:: ");
  scanf("%f %f", &a,&b);

  printf("The addition of %f and %f is:: %f\n", a,b,a+b);
  
  return 0;
}

File: cal.py
''' Basic Calculator in C 

Operations supported:
1. Addition
'''

if __name__=='__main__':
    a,b = raw_input('Input two numbers:: ').split()
    
    a = float(a)
    b = float(b)
    
    print 'The addition of {0} and {1} is {2}'.format(a,b,a+b)
